<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Principale_ihm</class>
 <widget class="QMainWindow" name="Principale_ihm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>950</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Jeu de Dame</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>img/icon.png</normaloff>img/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="Conteneur" native="true">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>800</width>
      <height>800</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>870</y>
      <width>691</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="bouton_new_2">
       <property name="text">
        <string>Nouvelle partie</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>238</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="bouton_qui">
       <property name="text">
        <string>Quitter</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>26</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>2</number>
   </property>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>Menu</string>
    </property>
    <widget class="QMenu" name="menuOptions">
     <property name="title">
      <string>Options</string>
     </property>
     <widget class="QMenu" name="menuJoueur_VS_IA">
      <property name="title">
       <string>Joueur VS IA</string>
      </property>
      <widget class="QMenu" name="menuNiveau_IA">
       <property name="title">
        <string>Niveau IA</string>
       </property>
       <addaction name="ialvl0"/>
       <addaction name="ialvl1"/>
       <addaction name="ialvl2"/>
       <addaction name="ialvl3"/>
       <addaction name="ialvl4"/>
       <addaction name="ialvl5"/>
      </widget>
      <widget class="QMenu" name="menuCouleur_joueur">
       <property name="title">
        <string>Couleur joueur</string>
       </property>
       <addaction name="sideBlanc"/>
       <addaction name="sideNoir"/>
       <addaction name="sideAlea"/>
      </widget>
      <addaction name="menuNiveau_IA"/>
      <addaction name="menuCouleur_joueur"/>
     </widget>
     <addaction name="playerVplayer"/>
     <addaction name="menuJoueur_VS_IA"/>
    </widget>
    <addaction name="actionSauvegarder"/>
    <addaction name="actionCharger"/>
    <addaction name="separator"/>
    <addaction name="menuOptions"/>
    <addaction name="actionQuitter"/>
   </widget>
   <addaction name="menuFiles"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuitter">
   <property name="icon">
    <iconset>
     <normaloff>../../../../Images/Pellicule/EOOUMfxW4AAJght.jpg</normaloff>../../../../Images/Pellicule/EOOUMfxW4AAJght.jpg</iconset>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="ialvl0">
   <property name="text">
    <string>0</string>
   </property>
  </action>
  <action name="ialvl1">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="ialvl2">
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="ialvl3">
   <property name="text">
    <string>3</string>
   </property>
  </action>
  <action name="ialvl4">
   <property name="text">
    <string>4</string>
   </property>
  </action>
  <action name="ialvl5">
   <property name="text">
    <string>5</string>
   </property>
  </action>
  <action name="actionSauvegarder">
   <property name="text">
    <string>Sauvegarder</string>
   </property>
  </action>
  <action name="actionCharger">
   <property name="text">
    <string>Charger</string>
   </property>
  </action>
  <action name="playerVplayer">
   <property name="text">
    <string>Joueur VS joueur</string>
   </property>
  </action>
  <action name="sideBlanc">
   <property name="text">
    <string>Blanc</string>
   </property>
  </action>
  <action name="sideNoir">
   <property name="text">
    <string>Noir</string>
   </property>
  </action>
  <action name="sideAlea">
   <property name="text">
    <string>Al√©atoire</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuitter</sender>
   <signal>triggered()</signal>
   <receiver>Principale_ihm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bouton_qui</sender>
   <signal>clicked()</signal>
   <receiver>Principale_ihm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>769</x>
     <y>906</y>
    </hint>
    <hint type="destinationlabel">
     <x>878</x>
     <y>904</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
